h/static/scripts/directives/thread.coffee

```diff
@@ -137,6 +142,12 @@ thread = [
         else
           attrs.$removeClass COLLAPSED_CLASS
 
+      scope.$watch $parse(attrs.thread), (thread) ->
```

Comment:
I'm going to merge this, as it fixes some bugs currently in prod. I'm still a bit dubious about this pattern, though, as it seems to me like explicitly having to call `$digest` like this implies we're using the scope wrong. At the very least, I'd like to see a comment explaining why this `$watch` is necessary.